l2-contracts/script/GrantRoles.s.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+import { AccessManager } from "@openzeppelin/contracts/access/manager/AccessManager.sol";
+import { Script } from "forge-std/Script.sol";
+import { console } from "forge-std/console.sol";
+
+/**
+ * @title Grant Roles Script
+ * @author Puffer Finance
+ */
+contract GrantRoles is Script {
+    // Role IDs
+    uint64 constant MERKLE_ROOT_POSTER_ROLE = 1;
+    uint64 constant MERKLE_ROOT_CANCELLER_ROLE = 2;
+    uint64 constant FUNDS_RESCUER_ROLE = 3;
+
+    function grantMerkleRootPosterRole(address accessManager) external {
+        // Read from .env or command line arguments
+        address posterAddress;
+        try vm.envAddress("MERKLE_ROOT_POSTER") returns (address envAddress) {
+            posterAddress = envAddress;
+        } catch {
+            posterAddress = msg.sender;
+        }
+
+        uint256 deployerPk = vm.envUint("PK");
```

Comment:
We should get rid of this way of doing things. 
This requires the caller to expose his secret key in env variable. If we just use `vm.startBroadcast()` then they can use keystore from foundry `--account` or `--private-key` or some other wallet options https://book.getfoundry.sh/reference/forge/forge-script#wallet-options---raw